perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/GroupMoveNotAllowedException.java

```diff
@@ -0,0 +1,24 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+/**
+ * Exception thrown when the group cannot be moved, because it's not allowed
+ * f.e. Destination group is subGroup of Moving group, Moving group is already in destination group as subGroup...
+ *
+ * @author Peter Balcirak
+ */
+public class GroupMoveNotAllowedException extends PerunException {
```

Comment:
Can you please update this exception to contain ``movingGroup`` and ``destinationGroup`` objects ? I can later use them in GUI to display nice message and group names. Right now exception message contains only serialized object.